quantum/quantum_keycodes.h

```diff
@@ -461,6 +461,7 @@ enum quantum_keycodes {
     UNICODE_MODE_REVERSE,
     UNICODE_MODE_OSX,
     UNICODE_MODE_LNX,
+    UNICODE_MODE_EMACS,
```

Comment:
This should actually be placed right before `SAFE_RANGE` at Line 517 so as to not break compatibility with VIA.

Reply:
Given that the only issue is switching a line, and even though @jannikadon didn't respond, should this pull request really have been closed as stale?  If there's a procedural issue, if I reopen this this PR with the same changes (modulo putting `UNICODE_MODE_EMACS` right before `SAFE_RANGE`), could this move ahead?